app/src/main/java/woowacourse/shopping/cart/viewHolder/CartNavigationViewHolder.kt

```diff
@@ -0,0 +1,40 @@
+package woowacourse.shopping.cart.viewHolder
+
+import android.view.LayoutInflater
+import android.view.View.GONE
+import android.view.View.VISIBLE
+import android.view.ViewGroup
+import woowacourse.shopping.databinding.CartNavigationItemBinding
+import woowacourse.shopping.model.CartUIModel
+
+class CartNavigationViewHolder private constructor(
+    private val binding: CartNavigationItemBinding,
+    onPageUp: () -> Unit,
+    onPageDown: () -> Unit
+) : ItemViewHolder(binding.root) {
+    init {
+        binding.pageUp.setOnClickListener { onPageUp() }
+        binding.pageDown.setOnClickListener { onPageDown() }
+    }
+
+    fun bind(cart: CartUIModel) {
+        binding.cartNavigation.visibility = when {
+            cart.pageUp -> VISIBLE
+            cart.pageDown -> VISIBLE
+            else -> GONE
+        }
```

Comment:
ViewHolder 설계가 아주 깔끔하네요 👍  when 문은 다음과 같이 작성해볼 수 있을 것 같아요.
```suggestion
        binding.cartNavigation.visibility = when {
            cart.pageUp, cart.pageDown -> VISIBLE
            else -> GONE
        }
```